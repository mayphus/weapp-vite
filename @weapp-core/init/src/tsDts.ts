export function getDefaultTsDts() {
  return `/// <reference types="weapp-vite/client" />
`
}
