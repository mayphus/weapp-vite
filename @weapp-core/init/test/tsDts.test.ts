import { getDefaultTsDts } from '@/tsDts'

describe('tsDts', () => {
  it('getDefaultTsDts', () => {
    expect(getDefaultTsDts()).toMatchSnapshot()
  })
})
