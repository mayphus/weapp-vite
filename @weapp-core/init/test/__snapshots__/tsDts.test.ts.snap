// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`tsDts > getDefaultTsDts 1`] = `
"/// <reference types="weapp-vite/client" />
"
`;
